node/language-translation/src/main.js

```diff
@@ -0,0 +1,40 @@
+import { HfInference } from '@huggingface/inference';
+import { getStaticFile, throwIfMissing } from './utils.js';
+
+export default async ({ req, res }) => {
+  throwIfMissing(process.env, ['HUGGINGFACE_ACCESS_TOKEN']);
+
+  if (req.method === 'GET') {
+    return res.send(getStaticFile('index.html'), 200, {
+      'Content-Type': 'text/html; charset=utf-8',
+    });
+  }
+
+  if (!req.body.sourceText && typeof req.body.sourceText !== 'string') {
+    return res.json({ ok: false, error: 'sourceText is required.' }, 400);
+  }
```

Comment:
Why don't we have an explicit condition to check for POST request? In this case, any non GET requests will enter this part of the code